lib/src/content/components/main_content/wolt_modal_sheet_main_content.dart

```diff
@@ -47,79 +35,55 @@ class _WoltModalSheetMainContentState extends State<WoltModalSheetMainContent> {
         ScrollController(initialScrollOffset: widget.currentScrollPosition.value);
   }
 
-  static const _defaultTopBarHeight = 56.0;
-
-  EdgeInsetsDirectional get _mainContentPadding {
-    final mainContentPadding = widget.page.mainContentPadding;
-    if (mainContentPadding != null) {
-      return mainContentPadding;
-    }
-    switch (widget.woltModalType) {
-      case WoltModalType.bottomSheet:
-        return const EdgeInsetsDirectional.all(16);
-      case WoltModalType.dialog:
-        return const EdgeInsetsDirectional.all(32);
-    }
-  }
-
   @override
   Widget build(BuildContext context) {
-    final heroImageHeight = widget.page.heroImageHeight ?? 0;
+    final page = widget.page;
+    final heroImageHeight = page.heroImage == null ? 0.0 : (widget.page.heroImageHeight ?? 0.0);
+    final pageHasTopBarLayer = page.hasTopBarLayer;
+    final navigationBarHeight = page.navigationBarHeight;
```

Comment:
```suggestion
    final pageHasTopBarLayer = page.hasTopBarLayer;
    final navigationBarHeight = page.navigationBarHeight;
```

I noticed these variables before in the package.

Here `pageHasTopBarLayer`  is only used once below. Why would you put in a variable?  

It makes sense to cache the value in the variable if it is to be used in multiple places in the class and to compare existing value when updating.

Reply:
>Here pageHasTopBarLayer is only used once below. Why would you put in a variable?

Ok, good point. You will see why I did it in my next PR where I will introduce the ThemeExtensions :) I wanted to reduce the diff already now.